ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -346,10 +346,19 @@ export default class ConfirmTransactionBase extends Component {
       </div>
     ) : null;
 
+    const showInlineControls = process.env.SHOW_EIP_1559_UI
```

Comment:
I'm losing my mind, I thought I did this already in #11481 -- but apparently this change got lost in the rebase? 

anyways this all looks fine but we should probably remove the work I did there in transaction-detail-item if its not needed now

Reply:
Gah, sorry!  I thought so as well but didn't think to look in the individual `item` component -- sorry!

Reply:
No worries the rebase issue was most likely my fault :D